packages/common/driver-definitions/src/driverError.ts

```diff
@@ -3,11 +3,103 @@
  * Licensed under the MIT License.
  */
 
+import { FluidErrorTypes } from "@fluidframework/core-interfaces";
+
 import { IResolvedUrl } from "./urlResolver";
 
+/**
+ * Different error types the Driver may report out to the Host.
+ */
+export const DriverErrorTypes = {
+	...FluidErrorTypes,
```

Comment:
Not sure about this...

Reply:
Can you elaborate? This was the pattern we adopted for `container-definitions` in #16868, for reference.

Reply:
Yeah thanks for checking.  It was about pulling in all of `FluidErrorTypes`.  There was another thread about this, it looks like you landed on pulling in everything by DPE/DCE which seems fine